x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/api/review_rule_upgrade/review_rule_upgrade_route.ts

```diff
@@ -92,6 +92,7 @@ const calculateRuleInfos = (results: CalculateRuleDiffResult[]): RuleUpgradeInfo
     return {
       id: installedCurrentVersion.id,
       rule_id: installedCurrentVersion.rule_id,
+      revision: installedCurrentVersion.revision,
```

Comment:
@xcrzx The `RuleUpgradeSpecifier` expects to be sent the current `revision` of the rule, but the response payload as it was did not return it. I added it here so it can be resent to the backend on rule upgrade _perform.
Does this make sense to you? Or did I misunderstand the OCC mechanism?

Reply:
Yup, that's correct. I overlooked the `revision` in my previous PR. Thank you for adding it, @jpdjere. 👍